lib/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_hermes_library(hermesOptimizer
   Optimizer/Scalar/InstSimplify.cpp
   Optimizer/Scalar/Auditor.cpp
   Optimizer/Wasm/WasmSimplify.cpp
+  Optimizer/Wasm/EmitWasmIntrinsics.cpp
```

Comment:
If we are moving EmitWasmIntrinsics into the optimizer lib, it should be removed from the frontend lib below.